Resources/views/line/video.html.twig

```diff
@@ -0,0 +1,27 @@
+{% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
```

Comment:
We don't have such extension in Twig. The only way to bypass this issue is to add script on every template.